cmd/buildah/from.go

```diff
@@ -21,30 +31,20 @@ var (
 		},
 		cli.BoolFlag{
 			Name:  "pull-always",
-			Usage: "pull the image even if one with the same name is already present",
+			Usage: "pull the image even if one with the same name is already present (supercedes pull option)",
```

Comment:
Thought these lines were supposed to be < 80 char.  The 80th char is the first 'l' in pull.  I'm personally fine with out adding this verbiage.  But if you think it's important to have that in, perhaps:
"pull the image even if the image is already present (supersedes pull option)"

Either way, please touch up supercedes to supersedes.

Reply:
Reworded to fix within 80 chars.
			Usage: "pull the image even if named image is present in store (supercedes pull option)",

Reply:
supersedes not supercedes right?  The new verbiage looks good to me otherwise.  FWIW. supercede is a go to typo/mis-spelling for me too.